src/Commands/Purview/GetRetentionLabel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using PnP.PowerShell.Commands.Base;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+using PnP.PowerShell.Commands.Utilities.REST;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace PnP.PowerShell.Commands.Purview
+{
+    [Cmdlet(VerbsCommon.Get, "PnPRetentionLabel")]
+    [OutputType(typeof(IEnumerable<Model.Graph.Purview.RetentionLabel>))]
+    [OutputType(typeof(Model.Graph.Purview.RetentionLabel))]
+    public class GetAvailableRetentionLabel : PnPGraphCmdlet
```

Comment:
Would also need `RecordsManagement.Read.All` as minimum permissions ? Can you please add that ? You can check other graph cmdlets where we have added different permissions.

If any issue, let me know :)

Reply:
This is great, thanks !!

Also, sorry, I wasn't clear. I meant add this attribute
`[RequiredMinimalApiPermissions("RecordsManagement.Read.All )]`

You can check `Add-PnPAzureADGroupMember` cmdlet for example 😊

Reply:
Oops, misunderstood the requirement. Thanks for handling it. Thanks very much for merging it. I will add the other operations, to add, delete and update the retention label.